modules/create-object-collections.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * operating/create-use-collections.adoc
+:_content-type: PROCEDURE
+[id="creating-object-collections_{context}"]
+= Creating object collections
+
+To access collections, click *Platform Configuration* -> *Collections*. The page displays a list of currently-configured collections. You can perform the following actions:
+
+* Search for collections by entering text in the *Search by name* field, and then press *->*. 
+* Click on a collection in the collection list to view the collection in read-only mode.
+* Click {kebab} for an existing collection to edit, clone, or delete it.
++
+[NOTE]
+====
+You cannot delete a collection that is actively used in {product-title-short}. ##I tried testing this, but I didn't get an error message and it allowed me to delete a collection that I'd used in a vulnerability report. Using stagingdb.demo.stackrox.com##
+====
+
+.Procedure
+To create an object collection:
+
+. In the Collections page, click *Create collection*.
+. Enter the name and description for the collection.
+. In the *Collection rules* section, you must define the rules for the collection. The results of your choices in this section are displayed in the *Collection results* live preview panel. Click *Hide results* to remove this panel from display. 
+. Optional: Add additional collections to a collection. Grouping collections and adding them to other collections can be useful if you want to create small collections based on deployments, then reuse and combine those smaller collections into larger, hierarchical collections. To add additional collections to a new collection that you are creating:
```

Comment:
Technically both attaching nested collections and configuring collection rules are both optional, though one is required at least. I don't know if there's an easier way to represent that in the docs but this is a place where potentially more clarity could be given on that.
ie.
One Of:
* Define a selector rule
* Attach another collection